### ISSUE TITLE
The query-panel block cannot be accessed via wp.blocks

### ISSUE THREAD

**Issue by amdrew** [WordPress/gutenberg#4998]
## Issue Overview

The `query-panel` block cannot be accessed via `wp.blocks`.

## Steps to Reproduce

```
const QueryPanel = wp.blocks.QueryPanel;
console.log(QueryPanel);
```

Will be `undefined`.

**Comment by amdrew** [comment]
PR: https://github.com/WordPress/gutenberg/pull/4999

**Comment by youknowriad** [comment]
I don't know this component well to tell if it's something generic enough and usefull enough to be exposed externally. And I agree with the comment in #4999 the name is misleading.

cc @jorgefilipecosta thoughts?

**Comment by jorgefilipecosta** [comment]
Hi @amdrew and @youknowriad, I shared my thougths in PR https://github.com/WordPress/gutenberg/pull/4999.

**Comment by gziolo** [comment]
This is going to be addressed with #5319.

**Comment by gziolo** [comment]
@amdrew I have just merged changes which expose `wp.components.QueryControls`. Let us know if it works for you.
